lnrpc/routerrpc/router_backend.go

```diff
@@ -330,7 +330,7 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 	// Query the channel router for a possible path to the destination that
 	// can carry `in.Amt` satoshis _including_ the total fee required on
 	// the route.
-	route, _, err := r.FindRoute(
+	route, successProb, err := r.FindRoute(
```

Comment:
The `fixup` commit needs to be squashed.

Reply:
Was not sure if I had to revert it, but it's squashed now.